src/main/resources/schemas/nanopore-instrument-output.schema.json

```diff
@@ -24,7 +24,7 @@
           "minLength": 1
         },
         "children": {
-          "description": "Describes one or more files and/or sub-folders",
+          "description": "Describes files and/or sub-folders if existant",
```

Comment:
existent*

Reply:
```suggestion
          "description": "Describes files and/or sub-folders if existent",
```

Reply:
you can make a suggestion, that ca be merged so you get contribution